SPECS/xcp-ng-xapi-plugins.spec

```diff
@@ -32,6 +32,13 @@ install SOURCES/etc/xapi.d/plugins/xcpngutils/*.py %{buildroot}/etc/xapi.d/plugi
 %dir /var/lib/xcp-ng-xapi-plugins
 
 %changelog
+* Fri Dec 17 2021 Benjamin Reis <benjamin.reis@vates.fr> - 1.7.0-1
+- Normalize plugins output
+- Add plugin to display return of lsblk
+- Fix lock in updater plugin
```

Comment:
That's the most important, I'd put it first. You can also reference https://github.com/xcp-ng/xcp/issues/523